test/runtests.jl

```diff
@@ -438,6 +461,15 @@ end
             end
         end
 
+        @testset "check that we can define custom hash function" begin
+            @auto_hash_equals runtests.myhash struct S470
```

Comment:
Curious, what happens if we use another macro in lieu of `runtests.myhash` here? Maybe that modifies the struct too?

Reply:
You would put the macro before `struct`, not before the name of the hash function.  There are two issues for supporting interoperation with other macros.  See #3 and #11.